examples/00_HH_simulator.ipynb

```diff
@@ -43,7 +45,13 @@
     "\n",
     "# sbi\n",
     "from sbi import utils as utils\n",
-    "from sbi.inference.base import infer"
+    "#from sbi.inference.base import infer\n",
```

Comment:
Remove the comment, or?

Reply:
yes well spotted - I'll remove the line completely